applications/common/frontend/src/components/ContactForm/contactForm.js

```diff
@@ -18,25 +18,62 @@ const initForm = () => {
     document.addEventListener(window.KYANITE_ON_DOM_CONTENT_LOAD, () => {
         Array.from(document.getElementsByClassName("contact-form"))
         .forEach(function(contactForm) {
+
+            const clsFormSubmitted = 'form-submitted';
+            const clsFormSubmitSuccess = 'form-success';
+            const clsFormSubmitFailure = 'form-failure';
+
+            const addFormClass = (clsName) => {
+                !contactForm.classList.contains(clsName) && contactForm.classList.add(clsName);
+            }
+
+            const removeFormClass = (clsName) => {
+                contactForm.classList.contains(clsName) && contactForm.classList.remove(clsName);
+            }
+
+            const toggleFormClass = (clsName, isAdd) => {
```

Comment:
Maybe it could be `add` or `shouldAdd`? When I saw `is` I was expecting a past form (`isAdded`) and reversed check: `isAdded ? removeFormClass(clsName) : addFormClass(clsName);`